### PR TITLE
Quick launch

### PR DESCRIPTION
**Pull request by Jonahss**
Added quickLaunch, which is used for testing in appium-ios-simulator
It't the simplest way to start a simulator. uses instruments which is why I put it here.

Can move it to appium-ios-simulator if it makes more sense there, but I put it here because the Instruments cli is being used.

### CODE REVIEW

**Comment by Jonahss** [comment]
@sebv

**Comment by sebv** [comment]
Merged in beta branch, could not get quicklaunch test to work so skipped it.

File: lib/utils.js [hunk 1]
```diff
@@ -129,6 +130,25 @@ async function getIwdPath(xcodeVersion) {
...
 }
 
+// this function launches an instruments test with a default test
+// that immediately passes. In this way we can start a simulator
+// and be notified when it completely launches
+async function quickLaunch (udid) {
+  let traceTemplatePath = await getXcodeTraceTemplatePath();
+  let appPath = path.resolve(rootDir, 'node_modules', 'ios-test-app', testAppPath[1]);
+  let scriptPath = path.resolve('../assets/blank_instruments_test.js');
+  let traceDocument = path.resolve('/', 'tmp', 'testTrace.trace');
```
Comment (jlipps) [inline lib/utils.js]: are we sure we want to hard-code these?

File: test/e2e/basics-e2e-specs.js [hunk 2]
```diff
@@ -17,12 +18,12 @@ chai.use(chaiAsPromised);
...
 var TEMP_DIR = path.resolve(__dirname, 'tmp');
 
-describe('intruments tests', function () {
+describe('instruments tests', function () {
   this.timeout(90000);
 
   async function newInstrument(opts) {
     _.extend(opts, {
-      app: path.resolve(rootDir, 'test/assets/TestApp.app'),
+      app: path.resolve(__dirname, '../../..', 'node_modules', 'ios-test-app', testAppPath[1]),
```
Comment (jlipps) [inline test/e2e/basics-e2e-specs.js]: seems like we're using path.resolve inconsistently, even within this one call. would like to see either:

```js
path.resolve(__dirname, '..', '..', '..', 'node_modules', 'ios-test-app', testAppPath[1]);
```

or

```
path.resolve(__dirname, '../../../node_modules/ios-test-app', testAppPath[1]);
```

same goes for previous use of this function

**Comment by jlipps** [comment]
one comment, then LGTM. realize this has already been merged

**Pull request closed without merging**